src/opencage.rs

```diff
@@ -608,8 +606,20 @@ pub struct Status {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Timestamp {
     pub created_http: String,
-    #[serde(deserialize_with = "from_ts")]
-    pub created_unix: NaiveDateTime,
+    pub created_unix: UnixTime,
```

Comment:
I think many projects are switching to the `time` crate.

Reply:
exactly, that's why I created #57